metamath.c

```diff
@@ -2130,7 +2130,7 @@ void command(int argc, char *argv[]) {
       }
 
       if (2/*error*/ == readTexDefs(0 /* 1 = check errors only */,
-          0 /* 1 = no GIF file existence check */ )) {
+          1 /* 1 = GIF file existence check */ )) {
```

Comment:
The comment should read 0 = ...

Reply:
No, the meaning of 1 and 0 have flipped here, so `1 = GIF file existence check` now instead of `1 = no GIF file existence check`, and to preserve behavior the actual argument has been changed from 0 to 1 (because in this case we want a GIF check).